app/controllers/forumthreads_controller.rb

```diff
@@ -80,6 +85,20 @@ def destroy
     redirect_to @thread.forum
   end
 
+  def search
+  end
+
+  def search_redirect
```

Comment:
In fact, there's no way of getting those 4 blank parameters out of the url without doing it manually. If someone wanted to share a link, it would take them some time to get the unneeded params out.